internal/pkg/metadb/blobref/chunkref/chunkref.go

```diff
@@ -21,15 +21,26 @@ import (
 	"cloud.google.com/go/datastore"
 	"github.com/google/uuid"
 	"github.com/googleforgames/open-saves/internal/pkg/cache"
+	"github.com/googleforgames/open-saves/internal/pkg/metadb/blobref"
 	"github.com/googleforgames/open-saves/internal/pkg/metadb/timestamps"
 )
 
+// ChunkRef is a metadata entity to keep track of chunks stored in an external blob store.
+// It is a child entity of and always associated to a BlobRef.
 type ChunkRef struct {
-	Key     uuid.UUID `datastore:"-"`
+	// Key is the primary key of the ChunkRef
```

Comment:
nit: comments on exported structs/fields should end with a period

Recommending adding a period to the fields in this struct